sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Release History
 
-## 1.0.0-beta.5 (Unreleased)
+## 3.0.0 (Unreleased)
+First stable release of the azure-ai-formrecognizer package. Targets the service's v2.0 API.
```

Comment:
do you plan to update this in a second PR once we get confirmation?

Reply:
I think I will keep this around until we get a confirmation. Don't want to merge this until we know for sure!